dependencies.py

```diff
@@ -55,6 +55,7 @@ def setup_requires():
     'jupyter_contrib_nbextensions': 'modular collection of jupyter extensions',
     'pillow': 'image library used for bitmap data functions',
     'dune-gdt~=2021.1,>=2021.1.2': 'generic discretization toolbox',
+    'dune-xt~=2021.1,>=2021.1.2': 'DUNE extensions for dune-gdt',
```

Comment:
https://zivgitlab.uni-muenster.de/pymor/pymor/-/pipelines/101098/failures
You didn't commit the result of running `dependencies.py`

Reply:
nice catch, thx

Reply:
Well, I missed this often enough myself to write an automatic check for it. So you're welcome :wink: